pyartcd/pyartcd/pipelines/scan_plashet_rpms.py

```
@@ -0,0 +1,389 @@
+                with koji_api.multicall(strict=True) as m:
+                    tasks = [
+                        m.queryHistory(
+                            table='tag_listing',
+                            tag=tag_name,
+                            afterEvent=plashet_event_id,
+                            queryOpts={'limit': 1, 'order': '-create_event'},
+                        )
+                        for tag_name in tag_names
+                    ]
+                histories = [task.result for task in tasks]
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)